Adjust paths in man pages to jive with our packages.

diff -uNr src/appl/bsd/klogind.M src/appl/bsd/klogind.M
--- src/appl/bsd/klogind.M	Fri Dec 17 15:43:48 1999
+++ src/appl/bsd/klogind.M	Wed Dec 22 14:29:20 1999
@@ -27,7 +27,7 @@
 the port indicated in /etc/inetd.conf.  A typical /etc/inetd.conf
 configuration line for \fIklogind\fP might be:
 
-klogin stream tcp nowait root /usr/cygnus/sbin/klogind klogind -e5c
+klogin stream tcp nowait root /usr/sbin/klogind klogind -e5c
 
 When a service request is received, the following protocol is initiated:
 
diff -uNr src/appl/bsd/kshd.M src/appl/bsd/kshd.M
--- src/appl/bsd/kshd.M	Fri Dec 17 15:43:49 1999
+++ src/appl/bsd/kshd.M	Wed Dec 22 14:29:47 1999
@@ -8,7 +8,7 @@
 .SH NAME
 kshd \- kerberized remote shell server
 .SH SYNOPSIS
-.B /usr/local/sbin/kshd -kr45ec
+.B /usr/sbin/kshd -kr45ec
 .SH DESCRIPTION
 .I Krshd
 is the server for the 
@@ -25,7 +25,7 @@
 on the port indicated in /etc/inetd.conf.  A typical /etc/inetd.conf
 configuration line for \fIkrshd\fP might be:
 
-kshell	stream	tcp	nowait	root	/usr/local/sbin/kshd	kshd -5c
+kshell	stream	tcp	nowait	root	/usr/sbin/kshd	kshd -5c
 
 When a service request is received, the following protocol is initiated:
 
diff -uNr src/appl/sample/sserver/sserver.M src/appl/sample/sserver/sserver.M
--- src/appl/sample/sserver/sserver.M	Fri Dec 17 15:44:10 1999
+++ src/appl/sample/sserver/sserver.M	Wed Dec 22 14:30:16 1999
@@ -59,7 +59,7 @@
 using a line in
 /etc/inetd.conf that looks like this:
 .PP
-sample  stream  tcp     nowait  root /usr/local/sbin/sserver	sserver
+sample  stream  tcp     nowait  root /usr/sbin/sserver	sserver
 .PP
 Since \fBsample\fP is normally not a port defined in /etc/services, you will
 usually have to add a line to /etc/services which looks like this:
diff -uNr src/appl/telnet/telnetd/telnetd.8 src/appl/telnet/telnetd/telnetd.8
--- src/appl/telnet/telnetd/telnetd.8	Fri Dec 17 15:44:26 1999
+++ src/appl/telnet/telnetd/telnetd.8	Wed Dec 22 14:30:47 1999
@@ -37,7 +37,7 @@
 .SM DARPA TELNET
 protocol server
 .SH SYNOPSIS
-.B /usr/libexec/telnetd
+.B /usr/sbin/telnetd
 [\fB\-a\fP \fIauthmode\fP] [\fB\-B\fP] [\fB\-D\fP] [\fIdebugmode\fP]
 [\fB\-edebug\fP] [\fB\-h\fP] [\fB\-I\fP\fIinitid\fP] [\fB\-l\fP]
 [\fB\-k\fP] [\fB\-n\fP] [\fB\-r\fP\fIlowpty-highpty\fP] [\fB\-s\fP]
diff -uNr src/config-files/kdc.conf.M src/config-files/kdc.conf.M
--- src/config-files/kdc.conf.M	Fri Dec 17 15:44:45 1999
+++ src/config-files/kdc.conf.M	Wed Dec 22 14:31:13 1999
@@ -139,7 +139,7 @@
 realm.
 
 .SH FILES 
-/usr/local/lib/krb5kdc/kdc.conf
+/etc/kerberos/krb5kdc/kdc.conf
 
 .SH SEE ALSO
 krb5.conf(5), krb5kdc(8)
diff -uNr src/kadmin/cli/kadmin.M src/kadmin/cli/kadmin.M
--- src/kadmin/cli/kadmin.M	Fri Dec 17 15:45:09 1999
+++ src/kadmin/cli/kadmin.M	Wed Dec 22 14:32:06 1999
@@ -722,9 +722,9 @@
 .RS
 .TP
 EXAMPLE:
-kadmin: ktremove -k /usr/local/var/krb5kdc/kadmind.keytab kadmin/admin
+kadmin: ktremove -k /etc/kerberos/krb5kdc/kadm5.keytab kadmin/admin
 Entry for principal kadmin/admin with kvno 3 removed
-	from keytab WRFILE:/usr/local/var/krb5kdc/kadmind.keytab.
+	from keytab WRFILE:/etc/kerberos/krb5kdc/kadm5.keytab.
 kadmin:
 .RE
 .fi
diff -uNr src/slave/kprop.M src/slave/kprop.M
--- src/slave/kprop.M	Fri Dec 17 15:48:53 1999
+++ src/slave/kprop.M	Wed Dec 22 14:35:45 1999
@@ -39,7 +39,7 @@
 This is done by transmitting the dumped database file to the slave
 server over an encrypted, secure channel.  The dump file must be created
 by kdb5_util, and is normally KPROP_DEFAULT_FILE
-(/usr/local/var/krb5kdc/slave_datatrans).
+(/etc/kerberos/krb5kdc/slave_datatrans).
 .SH OPTIONS
 .TP
 \fB\-r\fP \fIrealm\fP
@@ -51,7 +51,7 @@
 \fB\-f\fP \fIfile\fP
 specifies the filename where the dumped principal database file is to be
 found; by default the dumped database file is KPROP_DEFAULT_FILE
-(normally /usr/local/var/krb5kdc/slave_datatrans).
+(normally /etc/kerberos/slave_datatrans).
 .TP
 \fB\-P\fP \fIport\fP
 specifies the port to use to contact the
diff -uNr src/slave/kpropd.M src/slave/kpropd.M
--- src/slave/kpropd.M	Fri Dec 17 15:48:53 1999
+++ src/slave/kpropd.M	Wed Dec 22 14:38:57 1999
@@ -69,7 +69,7 @@
 This is done by adding a line to the inetd.conf file which looks like
 this:
 
-kprop	stream	tcp	nowait	root	/usr/local/sbin/kpropd	kpropd
+kprop	stream	tcp	nowait	root	/usr/sbin/kpropd	kpropd
 
 However, kpropd can also run as a standalone deamon, if the
 .B \-S
@@ -87,13 +87,13 @@
 \fB\-f\fP \fIfile\fP
 specifies the filename where the dumped principal database file is to be
 stored; by default the dumped database file is KPROPD_DEFAULT_FILE
-(normally /usr/local/var/krb5kdc/from_master).
+(normally /etc/kerberos/from_master).
 .TP
 .B \-p
 allows the user to specify the pathname to the
 .IR kdb5_util (8)
 program; by default the pathname used is KPROPD_DEFAULT_KDB5_UTIL
-(normally /usr/local/sbin/kdb5_util).
+(normally /usr/sbin/kdb5_util).
 .TP
 .B \-S
 turn on standalone mode.  Normally, kpropd is invoked out of
